powershell-analyzer/src/analyze.ps1

```diff
@@ -226,16 +229,21 @@ function Invoke-PopulateRulesArray {
     }
 }
 
+if (-not (Test-Path $Directory)) {
+    throw "Directory '$Directory' does not exist."
+}
+
```

Comment:
I really only wrote this so that I could write a Pester test, without having to think about the actual logic of this script 😆 .